setup.py

```diff
@@ -92,5 +92,8 @@ def parse_requirements():
         "galore": [
             "galore_torch",
         ],
+        "mosaicml-streaming": [
```

Comment:
[`question`](https://conventionalcomments.org/): `mosaicml-streaming` should be an optional dependency. Is this the right way of adding it in this capacity?